src/term/mod.rs

```diff
@@ -1085,28 +1085,28 @@ impl ansi::Handler for Term {
             ansi::Mode::SgrMouse => self.mode.remove(mode::SGR_MOUSE),
             ansi::Mode::LineWrap => self.mode.remove(mode::LINE_WRAP),
             _ => {
-                debug_println!(".. ignoring unset_mode");
+                debug!(".. ignoring unset_mode");
             }
         }
     }
 
     #[inline]
     fn set_scrolling_region(&mut self, region: Range<Line>) {
-        debug_println!("set scroll region: {:?}", region);
+        debug!("set scroll region: {:?}", region);
         self.scroll_region.start = min(region.start, self.grid.num_lines());
         self.scroll_region.end = min(region.end, self.grid.num_lines());
         self.goto(Line(0), Column(0));
     }
 
     #[inline]
     fn set_keypad_application_mode(&mut self) {
-        debug_println!("set mode::APP_KEYPAD");
+        debug!("set mode::APP_KEYPAD");
         self.mode.insert(mode::APP_KEYPAD);
     }
 
     #[inline]
     fn unset_keypad_application_mode(&mut self) {
-        debug_println!("unset mode::APP_KEYPAD");
+        debug!("unset mode::APP_KEYPAD");
```

Comment:
I think I can safely say that _all_ of the `debug_println!` occurrences here should be `trace!`. Come to think of it, the `[unhandled]` log statements in ansi.rs should probably be `debug!` instead of `warn!`.